models/Spaces.js

```diff
@@ -26,10 +25,7 @@ const SpaceSchema = Schema(
       type: String,
       required: false,
     },
-    participants: {
-      type: [Any],
-      required: false,
-    },
+    participants: [{ type: mongoose.Schema.Types.ObjectId, ref: 'User' }],
```

Comment:
lmao. Imagine nesting an entire model in here like you suggested instead of doing a single line of code XD